master/buildbot/test/integration/test_worker_comm.py

```diff
@@ -33,6 +36,8 @@
 from buildbot.util.eventual import eventually
 from buildbot.worker import manager as workermanager
```

Comment:
`mock`'s location sounds like a good place to be after this import (with an empty line in-between)